oxd-server/src/main/java/org/gluu/oxd/server/service/PublicOpKeyService.java

```diff
@@ -67,6 +76,55 @@ public RSAPublicKey getRSAPublicKey(String jwkSetUri, String keyId) {
         }
     }
 
+    public String getKeyId(Jwt idToken, String jwkSetUri, SignatureAlgorithm signatureAlgorithm, Use use) {
```

Comment:
It's redundant since `getRSAPublicKey` already identifies it. Please rename `getRSAPublicKey` to `getPublicKey`. We can have no only RSA.

Reply:
as discussed on chat, we need `getKeyId` method to pass the conformance test.
We have created `getPublicKey` method in `keyService` to get keys for `ECDSA` and `RSA`.